das/aggregator.go

```diff
@@ -345,3 +345,7 @@ func (a *Aggregator) String() string {
 func (a *Aggregator) HealthCheck(ctx context.Context) error {
 	return nil
 }
+
+func (a *Aggregator) ExpirationPolicy(ctx context.Context) arbstate.ExpirationPolicy {
+	return -1
```

Comment:
Rather than returning an invalid policy ID here, it seems better to query the aggregator's children and get their policies. If they all have the same policy, return that policy. Otherwise, return a new "MixedPolicy" type.